connexta/eve
# EVE 47: Preload pictures, cursor fix, month patch

#### What does this PR do?

Image loading is really slow on the DEV website, so images are loaded by React in the `componentDidMount()` function which will hopefully help.  Now there is only a pointer cursor when the media has a link, which makes the user experience a little clearer.  Lastly, I noticed the month in the event component was always one month behind.  I fixed that as well.

#### Who is reviewing it?

@vinamartin @bankent1 @alchucam @mcalcote @haydenudelson

#### How should this be tested?

`yarn go` and mouse over to make sure the pointer cursor appears only on the first image.  Make sure all images load properly.  Make sure month values in the event component are correct.

#### Screenshots

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
